includes/elements/container.php

```diff
@@ -409,7 +424,7 @@ protected function register_container_layout_controls() {
 			Group_Control_Flex_Container::get_type(),
 			[
 				'name' => 'flex',
-				'selector' => '{{WRAPPER}}',
+				'selector' => '{{WRAPPER}}, {{WRAPPER}} > .e-container__inner',
```

Comment:
Why do you need that? variables are scoped and inherited

Reply:
@StyleShit 

This is unnecessary indeed. I will restore the original version.